summoner-cli/src/Summoner/CLI.hs

```diff
@@ -108,6 +109,8 @@ runShow = \case
                 fetchedLicense <- fetchLicense licenseName
                 putTextLn $ unLicense fetchedLicense
   where
+    showAlignedBulletList :: (a -> [Text]) -> [a] -> IO()
+    showAlignedBulletList showT lst = mapM_ (infoMessage . T.append "➤ ") $ alignTable $ map showT lst
```

Comment:
Let's not have a separate `showAlignedBulletList`. It's quite tedious to maintain multiple formatting functions that do almost the same thing... Current `showBulletList` is generic enough to support this use case. Having the following:

```haskell
showGhcMeta :: GhcVer -> (Text, Text, Text)
alignTable :: [(Text, Text, Text)] -> [Text]
```

We can now write:

```haskell
showAlignedBulletList id $ alignTable $ map showGhcMeta $ reverse universe
```